packages/web/src/scss/components/Grid/index.html

```diff
@@ -69,6 +69,96 @@ <h2 class="docs-Heading">Responsive Columns</h2>
   </div>
 </section>
 
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">Custom Spacing</h2>
+  <div class="docs-Stack docs-Stack--start">
+
+    <div
+      class="Grid Grid--cols-2 Grid--tablet--cols-3 Grid--desktop--cols-4"
+      style="
+        --grid-spacing-y: var(--spirit-space-1000);
+        --grid-spacing-x: var(--spirit-space-1000);
+      "
+    >
+      <div class="docs-Box docs-Box--small">1/2, 1/3, 1/4</div>
+      <div class="docs-Box docs-Box--small">1/2, 1/3, 1/4</div>
+      <div class="docs-Box docs-Box--small">1/2, 1/3, 1/4</div>
+      <div class="docs-Box docs-Box--small">1/2, 1/3, 1/4</div>
+    </div>
+
+  </div>
+</section>
+
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">Responsive Custom Spacing</h2>
+  <div class="docs-Stack docs-Stack--start">
+
+    <div
+      class="Grid Grid--cols-2 Grid--tablet--cols-3 Grid--desktop--cols-4"
+      style="
+        --grid-spacing-y: var(--spirit-space-800);
+        --grid-spacing-y-tablet: var(--spirit-space-1000);
+        --grid-spacing-y-desktop: var(--spirit-space-0);
+        --grid-spacing-x: var(--spirit-space-1000);
+        --grid-spacing-x-tablet: var(--spirit-space-900);
+        --grid-spacing-x-desktop: var(--spirit-space-1200);
+      "
+    >
+      <div class="docs-Box docs-Box--small">1/2, 1/3, 1/4</div>
+      <div class="docs-Box docs-Box--small">1/2, 1/3, 1/4</div>
+      <div class="docs-Box docs-Box--small">1/2, 1/3, 1/4</div>
+      <div class="docs-Box docs-Box--small">1/2, 1/3, 1/4</div>
+    </div>
+
+  </div>
+</section>
+
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">Responsive Custom Vertical Spacing</h2>
```

Comment:
Please swap the horizontal and vertical examples so anything horizontal always goes first.